packages/react-components/src/gallery/dominantSpeaker.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { VideoGalleryRemoteParticipant } from '../types';
+
+/**
+ * Calculates the participants that should be renderd basis on the list of dominant speakers and currently renderd participants in a call.
+ * @param participants - Array containing all participants of a call. {@link @azure/communication-react#VideoGalleryRemoteParticipant}
+ * @param dominantSpeakers - An array containing the userId of dominant speakers in a call in the order of their dominance. 0th index is the most dominant, 1st is the second most etc
+ * @param visibleParticipants - Array containing currently rendered (visible) participants in the call. {@link @azure/communication-react#VideoGalleryRemoteParticipant}
```

Comment:
Would be nice to name this so that it's obvious that unlike `participants` and `dominantSpeakers`, this stores the *last* layout.

How about `previousParticipants`?

Reply:
It's not really previous participants. They are currently visible participants...